modules/labs-react/text-input/lib/TextInput.Field.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+
+import {borderRadius, inputColors, spaceNumbers, type} from '@workday/canvas-kit-react/tokens';
+import {
+  createComponent,
+  errorRing,
+  ErrorType,
+  ExtractProps,
+  styled,
+  StyledType,
+  Themeable,
+  useModelContext,
+} from '@workday/canvas-kit-react/common';
+import {Box} from '@workday/canvas-kit-labs-react/common';
+
+import {TextInputModelContext} from './TextInput';
+import {TextInputModel} from './useTextInputModel';
+
+export interface TextInputFieldProps extends ExtractProps<typeof Box, 'input'>, Themeable {
+  model?: TextInputModel;
+}
+
+const StyledBox = styled(Box)<
+  Pick<TextInputFieldProps, 'theme'> & Pick<TextInputModel['state'], 'hasError'> & StyledType
+>(
+  {
+    ...type.levels.subtext.large,
+    border: `1px solid ${inputColors.border}`,
+    display: 'block',
+    backgroundColor: inputColors.background,
+    borderRadius: borderRadius.m,
+    boxSizing: 'border-box',
+    height: 40,
+    transition: '0.2s box-shadow, 0.2s border-color',
+    padding: spaceNumbers.xxs, // Compensate for border
```

Comment:
Something we should think about and be careful of:

`Box` has `padding` on it, so `padding` is part of the accepted props. However, if a property is defined using `styled` or `css`, it will _always_ override a passed attribute. As it currently stands, the following will be allowed by Typescript, but not work at runtime:

```tsx
<TextInput.Field padding="l" />
```

The `padding` will be silently overridden by the `styled(Box)`. The following will work, however:

```tsx
<TextInput.Field css={{ padding: spacing.l }} />
```

Reply:
hmmm do we need to omit BoxProps if they are used in a base style?

Reply:
or actually should the style BoxProps get moved form styled down to TextInputField props

Reply:
went with css